File: app/src/main/java/com/example/weizhenbin/wangebug/modules/recreation/entity/JokeContentlistBaseBean.java
package com.example.weizhenbin.wangebug.modules.recreation.entity;

import com.chad.library.adapter.base.entity.MultiItemEntity;

/**
 * Created by weizhenbin on 2018/8/13.
 */

public class JokeContentlistBaseBean implements MultiItemEntity {
    private int type;


    public int getType() {
        return type;
    }

    public void setType(int type) {
        this.type = type;
    }

    @Override
    public int getItemType() {
        return type;
    }
}

File: app/src/main/java/com/example/weizhenbin/wangebug/modules/recreation/entity/JokeContentlistBaseBean.kt
package com.example.weizhenbin.wangebug.modules.recreation.entity

import com.chad.library.adapter.base.entity.MultiItemEntity

/**
 * Created by weizhenbin on 2018/8/13.
 */

open class JokeContentlistBaseBean : MultiItemEntity {


    var type: Int = 0

    override fun getItemType(): Int {

        return type
    }

    companion object {

        val TEXT = 1
        val PIC = 2
        val GIF = 3
    }
}
